ztp/source-crs/AmqSubscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: amq7-interconnect-subscription
+  namespace: amq-router
+spec:
+  channel: 1.10.x
+  name:  amq7-interconnect-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
Can you add a newline to keep github from flagging this.

Reply:
Done